Library/Homebrew/cmd/test-bot.rb

```diff
@@ -287,6 +298,9 @@ def brew_update
       if ENV["ghprbPullLink"]
         @url = ENV["ghprbPullLink"]
         @hash = nil
+      elsif ENV["TRAVIS_PULL_REQUEST"] && ENV["TRAVIS_PULL_REQUEST"] != "false"
+        @url = "https://github.com/#{ENV["TRAVIS_REPO_SLUG"]}/pull/#{ENV["TRAVIS_PULL_REQUEST"]}"
+        @hash = nil
```

Comment:
Do you intent to use Travis CI to test PR? I think it won't help much as we cannot use it generate bottles. Anyway your call.